content/docs/quickstart/_index.md

```diff
@@ -3,7 +3,13 @@ title: Get Started
 no_on_this_page: true
 ---
 
-Get started provisioning infrastructure with Pulumi.
+<div class="flex max-w-2xl">
+
+    <p class="text-xl leading-loose text-gray-600">
```

Comment:
Love the intro. 🎉  Could we leave the the sizing and color as-is for the moment?